### PR TITLE
[WIP] AAP-2079: Add AAP architectural diagram

### PR DESCRIPTION
**Pull request by lmalivert**
[APP-2079](https://issues.redhat.com/browse/AAP-2079) Need diagram in docs showing the ports required

Included architectural diagram in Chapter 4. Network ports and protocols.

Current status: Troubleshooting, image isn't appearing as it should.

### CODE REVIEW

File: downstream/assemblies/platform/assembly-network-ports-protocols.adoc [hunk 1]
```diff
@@ -7,6 +7,10 @@
 
 {PlatformName} (AAP) uses a number of ports to communicate with its services. These ports must be open and available for incoming connection to the {PlatformName} server in order for it to work. Ensure that these ports are available and are not being blocked by the server firewall.
 
+The following architectural diagram is an example of a fully deployed {PlatformNameShort} with all possible components.
+
+image::ansible-network-ports-protocols[Ansible Automation Platform Architectural Diagram]
```
Comment (ariordan-redhat) [inline downstream/assemblies/platform/assembly-network-ports-protocols.adoc:12]:
```suggestion
image::ansible-network-ports-protocols.png[Ansible Automation Platform Architectural Diagram]
```

Comment (lmalivert) [reply downstream/assemblies/platform/assembly-network-ports-protocols.adoc]: Thank you, @ariordan-redhat! Fixed.

**Review by sean-m-sullivan — APPROVED** [review]
LGTM

File: downstream/assemblies/platform/assembly-network-ports-protocols.adoc [hunk 2]
```diff
@@ -263,23 +267,23 @@ a|Open *only* if the internal database is used. Otherwise, this port should not
...
+|`cdn02.quay.io:443` | Access to container images provided by Red Hat and partners
+|`cdn03.quay.io:443` | Access to container images provided by Red Hat and partners
 |===
 
 [IMPORTANT]
 ====
-Image manifests and filesystem blobs are served directly from `registry.redhat.io`. 
-However, from 1st of May 2023, filesystem blobs are served from `quay.io` instead. 
+Image manifests and filesystem blobs are served directly from `registry.redhat.io`.
+However, from 1st of May 2023, filesystem blobs are served from `quay.io` instead.
```
Comment (sayjadha) [inline downstream/assemblies/platform/assembly-network-ports-protocols.adoc:279]: Per IBM style guide, do not use ordinal numerals (that is, numerals that use the abbreviations st, nd, rd, and th) to specify a date. See 'General guidance for dates' section in the IBM Style Guide (page 103). 
Suggest revising 1st of May 2023 -> 1 May 2023.

File: downstream/assemblies/platform/assembly-network-ports-protocols.adoc [hunk 3]
```diff
@@ -263,23 +267,23 @@ a|Open *only* if the internal database is used. Otherwise, this port should not
...
 To avoid problems pulling container images, you must enable outbound connections to the listed `quay.io` hostnames.
 
 This change should be made to any firewall configuration that specifically enables outbound connections to `registry.redhat.io`.
 
-Use the hostnames instead of IP addresses when configuring firewall rules. 
+Use the hostnames instead of IP addresses when configuring firewall rules.
 
-After making this change you can continue to pull images from `registry.redhat.io`. 
+After making this change you can continue to pull images from `registry.redhat.io`.
```
Comment (sayjadha) [inline downstream/assemblies/platform/assembly-network-ports-protocols.adoc:286]: Suggest "after making this change**,**" (add a comma after the verb phrase).

**Review by sayjadha — COMMENTED** [review]
Hi @lmalivert : Some minor edits, rest LGTM.

**Comment by sayjadha** [comment]
@lmalivert: Changes LGTM. Merging the PR.

**Comment by lmalivert** [comment]
Thank you, @sayjadha!

**Pull request merged**